common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -288,10 +303,27 @@ bool base_wlan_hal_dwpal::fsm_setup()
                     LOG(ERROR) << "Can't get event fd since dwpal ctx is NULL!";
                     return (transition.change_destination(dwpal_fsm_state::Detach));
                 }
-
```

Comment:
Separating blocks of code with spaces is actually a good idea. Makes the code easier to read.

Reply:
done